src/domain/Account.ts

```diff
@@ -10,22 +12,26 @@ export function createAccount(address: string): Account {
 }
 
 // used for determining nonZeroBalanceSince across system for user
-export function updateAccounts(event: TransferEvent): void {
-  // increase accountTo balance
-  let accountTo = Account.load(event.params.dst.toHex())
-  if (accountTo == null) {
-    accountTo = createAccount(event.params.dst.toHex())
+export function updateAccounts(event: TransferEvent, poolId: string): void {
```

Comment:
Can we give this function a name that better resembles the tying to the transfer event, e. g. `updateAccountsAfterTransfer`?